qlty-config/src/config/triage.rs

```diff
@@ -0,0 +1,165 @@
+use super::ignore::is_rule_issue_match;
+use super::plugin::PluginMode;
+use crate::config::issue_transformer::IssueTransformer;
+use globset::{Glob, GlobSet, GlobSetBuilder};
+use qlty_types::analysis::v1::{Category, Issue, Level};
+use qlty_types::{category_from_str, level_from_str};
+use schemars::JsonSchema;
```

Comment:
Can you please explain the motivation for introducing this schemars dependency?

Reply:
schemars was needed to allow adding the enums Category and Level instead if String for level and category attributes of Set. Since Set is part of Triage which is part of QltyConfig which implements JsonSchema. So Category and Level and everything in between also needs to implement JsonSchema

Reply:
Ah, I understand now. I had missed the fact that schemars was already in use.